.github/workflows/build-dist.yml

```diff
@@ -0,0 +1,164 @@
+name: build-dist
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    secrets:
+      GH_TOKEN:
+        description: Used by electron-publish\src\gitHubPublisher.ts
+        required: false
+      CSC_LINK:
+        description: Used to codesign binaries
+        required: false
+      CSC_KEY_PASSWORD:
+        description: Used to codesign binaries
+        required: false
+    inputs:
+      os:
+        required: true
+        type: string
+      node:
+        required: false
+        type: string
+        default: ${{ vars.DEFAULT_NODE_VERSION }}
+      max_build_attempts:
+        required: false
+        type: number
+        default: 5
+      cache_dir:
+        required: false
+        type: string
+        default: dist
+      npm_network_timeout:
+        required: false
+        type: number
+        default: 120000
+
+env:
+  # Disable console advertisements during install phase
+  ADBLOCK: true
+
+jobs:
+  build:
+    name: Build signed artifacts on ${{ inputs.os }}
+    runs-on: ${{ inputs.os }}
+
+    strategy:
+      fail-fast: false
+
+    concurrency:
+      group: '${{ github.workflow }}-${{ inputs.os }}-${{ inputs.node }}-${{ github.head_ref || github.ref_name }}'
+      cancel-in-progress: true
+
+    env:
+      USE_HARD_LINKS: false
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+
+      - name: Calculate build-cache and dist-cache keys
+        # Because the exact same hashFiles invocation produces different
+        # results down the file
+        id: build-cache-keys
+        shell: bash
+        run: |
+          if command -v shasum; then sum() { shasum -a 256 "$@"; } ; else sum() { sha256sum "$@"; }; fi
+          fragment="${{ runner.os }}-${{ inputs.node }}-${{ hashFiles('*.ts', '**/src/**', '**/.npmrc', '**/tsconfig*.json', '**/yarn.lock') }}-$(< package.json sum | command head -c8)"
```

Comment:
@strogonoff Sorry, I totally missed this...

Anyway, that was an artifact of my previous attempts at selecting attributes from `package.json` that I thought would influence a build, _i.e._ trying to make trigger more granular.  But I think that has since been removed.